<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="LeftToolBar" name="leftwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>500</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="headportrait">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/img/tx2.jpg</normaloff>:/images/img/tx2.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>设置</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#settings:!hover:!pressed{border-image:url(:/images/img/setting_01.png),30px,30px;
background-repeat:no-repeat;
background-position:center;} 
QPushButton#settings:hover{border-image:url(:/images/img/setting_02.png);
background-repeat:no-repeat;
background-position:center;}  
 
QPushButton#settings:pressed{border-image:url(:/images/img/setting_02.png);
background-repeat:no-repeat;
background-position:center;}

QPushButton#settings:released{border-image:url(:/images/img/setting_02.png);
background-repeat:no-repeat;
background-position:center;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_chatNow">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QpushButton#b_chatNow{
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/images/img/chatNow01.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_friendList">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>115</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/images/img/friend01.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="rightwidget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>550</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>500</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F5F5F5;  </string>
   </property>
   <widget class="QStackedWidget" name="rightStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>550</width>
      <height>460</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>460</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>16777215</height>
     </size>
    </property>
    <widget class="QWidget" name="messageBox">
     <widget class="QWidget" name="messageBoxWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>550</width>
        <height>460</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="msgContentEditArea">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>550</width>
         <height>100</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
background-color: #FCFCFC;
border: 1px solid #ECECEC;
padding: 8px 1em;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="sendMessageButton">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>426</y>
         <width>70</width>
         <height>28</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>28</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton#sendMessageButton:!hover:!pressed{
color: #606060;
background-color: #FFFFFF;
border: 1px solid #E1E1E1;
border-radius: 2px;
} 
QPushButton#sendMessageButton:hover{
color: #FFFFFF;
background-color: #3DCE3D;
border: 0px solid #FFFFFF;
border-radius: 2px;
}  
QPushButton#sendMessageButton:pressed{background-repeat:no-repeat;background-position:center;}
 </string>
       </property>
       <property name="text">
        <string>发送(S)</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_chatMsgContent">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>550</width>
         <height>290</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton#pushButton:!hover:!pressed{
background-image: url(:/images/img/emoji/smile.png);
background-repeat:no-repeat;
background-position:center;
border: 0px;
} 
QPushButton#pushButton:hover{
background-image: url(:/images/img/emoji/smile01.png);
background-repeat:no-repeat;
background-position:center;
border: 0px;
border: 0px solid #FFFFFF;

}  </string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="showInfo">
     <widget class="QWidget" name="showInfoWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>550</width>
        <height>460</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>550</width>
        <height>460</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="ListView" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>200</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>500</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #EAEAEA;  </string>
   </property>
   <widget class="QLineEdit" name="search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>200</width>
      <height>460</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="chat_now">
     <widget class="ListWidget_chatNow" name="listWidget_chatnow">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>460</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>460</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>460</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScrollMargin">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="friends">
     <widget class="ListWidget_friends" name="listWidget_friends">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>460</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>460</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>460</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="MainTitleWindow" name="ttitlewindow" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>550</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>40</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="titlelabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>微信</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>关闭</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#closeButton:!hover:!pressed{border-image:url(:/images/img/icon_close_gray_button.png);
background-repeat:no-repeat;
background-position:center;} 
QPushButton#closeButton:hover{border-image:url(:/images/img/icon_close_red_button.png);
background-repeat:no-repeat;
background-position:center;}  
QPushButton#closeButton:pressed{border-image:url(:/images/img/icon_close_red_button.png);
background-repeat:no-repeat;
background-position:center;}
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainTitleWindow</class>
   <extends>QWidget</extends>
   <header location="global">maintitlewindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LeftToolBar</class>
   <extends>QWidget</extends>
   <header location="global">lefttoolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListWidget_chatNow</class>
   <extends>QListWidget</extends>
   <header location="global">listwidget_chatnow.h</header>
  </customwidget>
  <customwidget>
   <class>ListWidget_friends</class>
   <extends>QListWidget</extends>
   <header location="global">listwidget_friends.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
